lib/workers/global/config/parse/cli.ts

```diff
@@ -34,6 +34,9 @@ export function getConfig(input: string[]): AllConfig {
         .replace('--aliases', '--registry-aliases')
         .replace('--include-forks=true', '--fork-processing=enabled')
         .replace('--include-forks', '--fork-processing=enabled')
+        .replace('--recreate-closed=false', '--recreate-closed=auto')
+        .replace('--recreate-closed=true', '--recreate-closed=always')
+        .replace(regEx(/--recreate-closed$/), '--recreate-closed=always')
```

Comment:
Why regex?

Reply:
Else we will replace the `--recreate-closed` in `--recreate-closed=xxx` with `always`

Reply:
None of the other ones ever needed it?

Reply:
Cause they were renames.

Reply:
We use regex with some 
```js
   .replace(/^--dry-run$/, '--dry-run=true')
        .replace(/^--require-config$/, '--require-config=true')
```